src/lm_datasets/datasets/multilingual/wura.py

```diff
@@ -0,0 +1,33 @@
+from lm_datasets.datasets.base import Availability, GB, License
+from lm_datasets.datasets.hf_dataset import HFDataset
+
+class WuraCorpus(HFDataset):
+    """
+    OVERLAP_WITH mc4
+    """
+    DATASET_ID = "wura"
+
+    TITLE = "WURA"
+    DESCRIPTION = (
+        """Wura is large-scale pretraining data for 20 languages popularly
+        spoken in Africa."""
+    )
+
+    HOMEPAGE = "https://huggingface.co/datasets/castorini/wura"
+    LICENSE = License(
+        name="Apache License Version 2.0",
+        url="http://www.apache.org/licenses/LICENSE-2.0"
+    )
+    AVAILIBILITY = Availability.DIRECT_DOWNLOAD
+
+    LANGUAGES = ["en", "fr", "pt", "af", "am", "ha", "ig", "rw", "mg", "ny",
```

Comment:
Can you create for each language a new dataset class? So you create one "base" clasas and then each language subclass inherits from it but you only change the dataset_id and the languages. Essentially like here https://github.com/malteos/lm-datasets/blob/main/src/lm_datasets/datasets/multilingual/macocu.py

Reply:
Should add them separeatly to the registry?
Or only the base Wura class?

Reply:
No please add all subclasses but not the base class.